app/src/main/java/eu/kanade/tachiyomi/ui/library/LibraryScreenModel.kt

```diff
@@ -281,7 +281,7 @@ class LibraryScreenModel(
         val sortFn: (LibraryItem, LibraryItem) -> Int = { i1, i2 ->
             val sort = keys.find { it.id == i1.libraryManga.category }!!.sort
             when (sort.type) {
-                LibrarySort.Type.Alphabetical -> {
+                LibrarySort.Type.Alphabetical, LibrarySort.Type.Random -> {
```

Comment:
I'm using alphabetical sort as a fallback so this function doesn't need to be nullable, this code path should never be accessed because in the random case the value map doesn't fetch a comparator.